index.html

```diff
@@ -2845,6 +2866,14 @@ <h3><dfn data-lt="creating a new session">New Session</dfn></h3>
     property</a> of <var>capabilities</var> with name
     "<code>pageLoadStrategy</code>" and value "<code>normal</code>".
 
+   <li><p>Let <var>strictFileInteractability</var> be the result of getting property
+    "<code>strictFileInteractability</code>" from <var>capabilities</var>.
+
+   <li><p>If <var>strictFileInteractability</var> is a boolean,
+    set the <a>current session</a>’s <a>strict file interactability</a> to
+    <var>strictFileInteractability</var>. Otherwise set the
+    <a>current session</a>'s <var>strict file interactability</a> to <code>false</code>.
```

Comment:
When we are deserializing the capabilities it looks like we error if this is not set as a boolean so we don't need the default though it makes me wonder if we should set a default if nothing comes over the wire earlier than here

Reply:
I think it can be not a boolean if the property isn't set at all in which case `getting a property` will return `undefined`.

Reply:
That makes sense.